gitlab.sh

```diff
@@ -1,28 +0,0 @@
-#!/bin/bash
```

Comment:
Not sure why GitHub shows this as a separate file from the old `gitlab.sh`. Any idea anyone?